### PR TITLE
Updating README and CONTRIBUTING [MAILPOET-1541]

### PR DESCRIPTION
**Pull request by amine-mp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,187 +1,130 @@
...
-  <%= partial('my_template_2', 'form/templates/blocks.hbs', '_my_partial') %>
-
-  <!-- custom partial using partial defined above -->
-  <script id="my_template_3" type="text/x-handlebars-template">
-    {{> _my_partial }}
-  </script>
-<% endblock %>
-```
-
-# i18n
```
Comment (pavel-mailpoet) [inline README.md]: I don't think we should delete all this information about translation functions. I think we should expand this section and explain what and when to use.

Comment (costasovo) [reply README.md]: I agree that we should keep the section about translation.

File: README.md [hunk 2]
```diff
@@ -1,187 +1,130 @@
...
-
-Run 'svn copy ...' to tag the release
-```
-It's quite literal: you can review the changes to be pushed and if you're satisfied, run the suggested command to finish the release publishing process.
-
-If you're confident, execute `./do publish --force` and your release will be published to the remote SVN repository without manual intervention (automatically). For easier authentication you might want to set `WP_SVN_USERNAME` and `WP_SVN_PASSWORD` environment variables.
-
-# Acceptance testing
+# Frameworks and libraries
+
```
Comment (costasovo) [inline README.md]: It is hard to pick important libraries and don't list all of them :). 
I would add also these to the list:
twig: templates rendering
monolog: logging
handlebars: templates for editor

File: README.md [hunk 3]
```diff
@@ -1,187 +1,130 @@
...
+## Installation
+```bash
+# go to WP plugins directory
+$ cd path_to_wp_directory/wp-content/plugins
+# clone this repository
+$ git clone https://github.com/mailpoet/mailpoet.git
+$ cd mailpoet
+# create the .env file
+$ cp .env.sample .env
+# change the values on .env file
```
Comment (costasovo) [inline README.md]: This step might be confusing. We require a lot of api keys in .env file but I think that some of them are not critical to be able to do some development. e.g. if developer doesn't have aws credentials the AWS SES unit test will fail.
Not everyone has an AWS account or Mailchimp account and I can't imagine that someone will create all these accounts (AWS requires also some configuration) during this step.
I suggest to divide .env.sample file by a comment line to required and optional settings and point out that without the optional settings some functions will not work and some tests will fail.
What do you think?

File: README.md [hunk 4]
```diff
@@ -1,187 +1,130 @@
...
+- [Codeception](https://codeception.com/) is used to write unit and acceptance tests.
+- [Docker](https://www.docker.com/), [Docker Compose](https://docs.docker.com/compose/) and [Selenium](https://www.seleniumhq.org/) to run acceptance tests.
+- [React](https://reactjs.org/) is used to create most of UIs.
+- [Marionette](https://marionettejs.com/) is used to build the newsletters editor.
+- [Stylus](http://stylus-lang.com/) is used to write styles.
+- [Mocha](https://mochajs.org/), [Chai](https://www.chaijs.com/) and [Sinon](https://sinonjs.org/) are used to write Javascript tests.
+- [ESLint](https://eslint.org/) is used to lint JS files.
+- [Webpack](https://webpack.js.org/) is used to bundle assets.
+
+# Files structure
```
Comment (pavel-mailpoet) [inline README.md]: Why do you think we need this section here? It will get outdated and I don't see the benefits.

File: README.md (hunk 3 above)
Comment (amine-mp) [reply README.md]: Yes, I agree. That's what I was discussing here https://mailpoet.slack.com/archives/G0A1L2KHV/p1539079892000100

I will edit `.env.sample` to make it clear.

File: README.md (hunk 4 above)
Comment (amine-mp) [reply README.md]: My idea was to show where each type of source code goes (PHP, JS and CSS). I think this global structure does not change that much to be outdated.

Comment (pavel-mailpoet) [reply README.md]: I understand, but everyone can find where each type of source goes. If you cannot find where PHP lives you probably shouldn't contribute. And I can see a couple of files that changed quite recently. Even the PHP version wasn't up-to-date so we cannot count on ourselves.

**Pull request merged**